### ISSUE TITLE
Skin to use setting reverts to "Main plugin folder - default-v4" without warning

### ISSUE THREAD

**Issue by ssteinerx** [marvinlabs/customer-area#127]
I set it to "Main plugin folder - default" and, for no apparent reason, it gets set back.

**Comment by vpratfr** [comment]
Hi,

would be nice to get some way to reproduct the issue. I never faced this issue till now.

**Comment by ssteinerx** [comment]
Agreed, but it seems 'random.'  I'm convinced it has to do with saving the settings, but I'm not sure which particular set triggers the issue.

I'll let you know if I can duplicate it, but so far, no luck.

S

**Comment by vpratfr** [comment]
Closing this as the skins have been updated. Re-open if that is happening post 7.0
